src/js/media-captions-button.ts

```diff
@@ -36,6 +41,11 @@ slotTemplate.innerHTML = /*html*/ `
   </slot>
 `;
 
+const tooltipContent = /*html*/ `
+  <slot name="tooltip-enable">Enable captions</slot>
+  <slot name="tooltip-disable">Disable captions</slot>
```

Comment:
(non-b) fine to iterate and add these later to a constants file for i18n etc

Reply:
Was chatting to Christian about this and have added these as constants to the labels util file now as a simple dictionary, as re-using the verbs dictionary wasn't quite correct and was missing entries